bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,304 @@
+# Babylon Validator Setup
+
+Before setting up a validator, you'll need:
+1. A fully synced Babylon node. For node setup instructions, see our 
+[Node Setup Guide](../babylon-node/README.md)
+2. Sufficient BBN tokens. For instructions on obtaining BBN tokens, see the 
+[Get Funds section](../babylon-node/README.md#get-funds) in the Node Setup Guide.
+
+## System Requirements
+
+Recommended specifications for running a Babylon validator node:
```

Comment:
Here, we should add a node that for RPC nodes, more RAM is required. Let's do it on the next iteration, once we have completed the RPC node load test.